packages/k/kitty/package.yml

```diff
@@ -24,6 +24,7 @@ builddeps  :
     - pkgconfig(xinerama)
     - pkgconfig(xkbcommon)
     - pkgconfig(xrandr)
+    - pkgconfig(libxxhash)
     - librsync-devel
     - golang
 rundeps    :
```

Comment:
If librsync changed to being opened dynamically with dlopen then `librsync` needs to be added as a rundep.

Reply:
@ReillyBrogan see my commit message